src/main/java/com/feedzai/commons/sql/abstraction/dml/result/SqlServerResultColumn.java

```diff
@@ -32,4 +34,27 @@
     public SqlServerResultColumn(final String name, final Object val) {
         super(name, val);
     }
+
+    @Override
+    public Boolean toBoolean() {
+        if (isNull()) {
+            return null;
+        }
+
+        if (super.val instanceof Boolean) {
+            return (Boolean) super.val;
+        }
+
+        final String val = super.val.toString();
+
+        if (val.equals("1")) {
+            return true;
+        }
+
+        if (val.equals("0")) {
+            return false;
+        }
+
+        throw new DatabaseEngineRuntimeException(val + " is not a boolean type");
```

Comment:
nitpick: I would prefer a if-then-else approach, but I'm nitpicking so it's just a suggestion. :)

Reply:
I'll leave it this way to keep consistency with the other `ResultColumn` implementations that already follow this pattern.